packages/ui-tabs/src/Tabs/styles.ts

```diff
@@ -116,7 +116,7 @@ const generateStyle = (
       })
     },
 
-    scrollOverlay: {
+    rightScrollOverlay: {
       label: 'tabs__scrollOverlay',
```

Comment:
The label and the property name have to match so it can be easily identifiable in the rendered code. Also, since we are writing bidirectional components, I would recommend naming them with "start" instead of "left" and "end" instead of "right".

```suggestion
    endScrollOverlay: {
      label: 'tabs__endScrollOverlay',
```